### PR TITLE
Add pagination to set_description_plaintext_task

### PR DESCRIPTION
**Pull request by 8r2y5**
I want to merge this change because it prevents infinite loop.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migrations are either absent or optimized for zero downtime
* [ ] The changes are covered by test cases

### CODE REVIEW

**Review by patrys — CHANGES_REQUESTED** [review]
It needs explicit ordering by PK to work. Otherwise, it could accidentally find the last product in the first batch.

File: saleor/product/migrations/tasks/saleor3_9.py [hunk 1]
```diff
@@ -17,14 +17,14 @@ def set_description_plaintext(qs: QuerySet[Product]):
...
+        Q(description_plaintext="", pk__gt=last_pk) & ~Q(description=None)
     ).order_by("-pk")
 
     ids = products.values_list("pk", flat=True)[:2000]
     qs = Product.objects.filter(pk__in=ids)
 
     if ids:
         set_description_plaintext(qs)
-        set_description_plaintext_task.delay()
+        set_description_plaintext_task.delay(qs.last().pk)
```
Comment (patrys) [inline saleor/product/migrations/tasks/saleor3_9.py:30]: Will `.last()` not result in a new DB call?

File: saleor/product/migrations/tasks/saleor3_9.py [hunk 2]
```diff
@@ -17,14 +17,14 @@ def set_description_plaintext(qs: QuerySet[Product]):
 
 
 @app.task
-def set_description_plaintext_task():
+def set_description_plaintext_task(last_pk: int = 0):
```
Comment (kadewu) [inline saleor/product/migrations/tasks/saleor3_9.py:20]: Don't we need to change the direction of ordering if we start from 0 and use `greater_than`?

File: saleor/product/migrations/tasks/saleor3_9.py (hunk 1 above)
Comment (kadewu) [inline saleor/product/migrations/tasks/saleor3_9.py:30]: And probably `ids[-1]` will be safer if applied what I mention above.

Comment (8r2y5) [reply saleor/product/migrations/tasks/saleor3_9.py:30]: It won't work as slices does not provide that functionality

Comment (patrys) [reply saleor/product/migrations/tasks/saleor3_9.py:30]: `max(ids)` should work, though, no? The QuerySet is already evaluated.

**Review by kadewu — APPROVED** [review]

**Review by patrys — APPROVED** [review]

**Pull request merged**